### PR TITLE
add v0.5.0 changelog

### PR DESCRIPTION
**Pull request by danpi**
### Motivation

Add v0.5.0 changelog

### Modifications

Add v0.5.0 changelog

### Verifying this change

- [x] Make sure that the change passes the CI checks.

- [x] This change is a trivial rework / code cleanup without any test coverage.

### Documentation

Check the box below.

Need to update docs? 

- [x] `no-need-doc`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+# Function Mesh v0.5.0 Release Notes
+
+## v0.5.0 What's New
+
+- [Test] Add tests for packages download ([#429](https://github.com/streamnative/function-mesh/pull/429))
+- [Test] refine test cases of the Sink/Source/Crypto function ([#435](https://github.com/streamnative/function-mesh/pull/435))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:6]:
```suggestion
- [Test] Refine the test cases of the sinks, sources, and Crypto functions ([#435](https://github.com/streamnative/function-mesh/pull/435))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+# Function Mesh v0.5.0 Release Notes
+
+## v0.5.0 What's New
+
+- [Test] Add tests for packages download ([#429](https://github.com/streamnative/function-mesh/pull/429))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:5]:
```suggestion
- [Test] Add tests for package download ([#429](https://github.com/streamnative/function-mesh/pull/429))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+- [Test] refine test cases of the Sink/Source/Crypto function ([#435](https://github.com/streamnative/function-mesh/pull/435))
+- [Test] Use v1.8.2 for cert-manager tests ([#433](https://github.com/streamnative/function-mesh/pull/433))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:7]:
```suggestion
- [Test] Use cert-manager v1.8.2 for tests ([#433](https://github.com/streamnative/function-mesh/pull/433))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
+- [Helm Charts] Add handle logic for FunctionMesh to clean up orphaned subcomponents ([#431](https://github.com/streamnative/function-mesh/pull/431))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:8]:
```suggestion
- [Helm Charts] Add the handle logic for Function Mesh to clean up orphaned subcomponents ([#431](https://github.com/streamnative/function-mesh/pull/431))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,19 @@
...
+- [Controller] Add the ability to set logging levels ([#445](https://github.com/streamnative/function-mesh/pull/445))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:9]:
```suggestion
- [Controller] Support setting the logging levels ([#445](https://github.com/streamnative/function-mesh/pull/445))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,19 @@
...
+- [Controller] Update CSV for openshift ([#417](https://github.com/streamnative/function-mesh/pull/417))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:10]:
```suggestion
- [Controller] Update the CSV for OpenShift ([#417](https://github.com/streamnative/function-mesh/pull/417))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,19 @@
...
+- [Controller] Add sky-walking workflow for self-signed tls pulsar cluster ([#439](https://github.com/streamnative/function-mesh/pull/439))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:11]:
```suggestion
- [Controller] Add the sky-walking workflow for the Pulsar cluster enabled with the self-signed TLS certificate ([#439](https://github.com/streamnative/function-mesh/pull/439))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,19 @@
...
+- [Controller] Add a limit to the resource name ([#437](https://github.com/streamnative/function-mesh/pull/437/files))
+- [Function Mesh Worker service] Support subscriptionPosition for function&sink ([#322](https://github.com/streamnative/function-mesh-worker-service/pull/322))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:13]:
```suggestion
- [Function Mesh Worker service] Support the `subscriptionPosition` for functions and sinks ([#322](https://github.com/streamnative/function-mesh-worker-service/pull/322))
```

**Review by Huanli-Meng — APPROVED** [review]
leave some editorial review comments.

**Review by freeznet — APPROVED** [review]

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,19 @@
...
+- [Test] Refine the test cases of the sinks, sources, and Crypto functions ([#435](https://github.com/streamnative/function-mesh/pull/435))
+- [Test] Use cert-manager v1.8.2 for tests ([#433](https://github.com/streamnative/function-mesh/pull/433))
+- [Helm Charts] Add the handle logic for Function Mesh to clean up orphaned subcomponents ([#431](https://github.com/streamnative/function-mesh/pull/431))
+- [Controller] Support setting the logging levels ([#445](https://github.com/streamnative/function-mesh/pull/445))
+- [Controller] Update the CSV for OpenShift ([#417](https://github.com/streamnative/function-mesh/pull/417))
+- [Controller] Add the sky-walking workflow for the Pulsar cluster enabled with the self-signed TLS certificate ([#439](https://github.com/streamnative/function-mesh/pull/439))
+- [Controller] Add a limit to the resource name ([#437](https://github.com/streamnative/function-mesh/pull/437/files))
+- [Function Mesh Worker service] Support the `subscriptionPosition` for functions and sinks ([#322](https://github.com/streamnative/function-mesh-worker-service/pull/322))
```
Comment (freeznet) [inline CHANGELOG.md:13]: please remove the mesh worker service changes in function-mesh's changelog.
```suggestion
```

**Review by freeznet — APPROVED** [review]

**Review by freeznet — APPROVED** [review]

**Comment by danpi** [comment]
> 

thanks a lot for the suggestions, all done.

**Review by danpi — COMMENTED** [review]
thanks a lot for the suggestions, all done.

**Review by tpiperatgod — APPROVED** [review]
lgtm

**Pull request merged**